### DISCUSSION TITLE
Payment routing issues.

### DISCUSSION THREAD

**Discussion by 17621859515** [lightningnetwork/lnd#7327] [Q&A]
My link path is
my node -> 038863cf8ab91046230f561cd5b386cbff8309fa02e3f0c3ed161a3aeb64a643b9@203.132.94.196:9735

My payment path is
my node -> 203.132.94.196:9735 -> https://htlc.me/
Successfully paid 0.0001

now i think
https://htlc.me/ -> my node

Pay 0.0001
unable to succeed
What is the reason?

![image](https://user-images.githubusercontent.com/69946784/212663792-6d4007f1-7107-4ae0-9dcd-13dfae03d160.png)

**Comment by 17621859515** [comment]
can someone help me？

**Reply by bitromortac** [reply]
It's hard to say why your payment failed as the payer in this case is htlc.me. In general, you need to have enough inbound liquidity. Also, routing on testnet may be degraded, so it's important to have good peers.

**Reply by 17621859515** [reply]
So how should I increase the liquidity of my local wallet to ensure that I can effectively receive btc?
At present, my local wallet can only actively link to public nodes, but cannot be deposited into btc

**Reply by bitromortac** [reply]
I would say that tools to acquire inbound liquidity are better on mainnet, you could use loop out for example (I don't know how well it works on testnet). You can also try to "buy" something here https://starblocks.acinq.co/ to get inbound liquidity. Another way would be to open a channel with a push amount (don't do this on mainnet as you'll gift/sats to your peer).
